### ISSUE TITLE
TypeError: 'NoneType' object is not iterable

### ISSUE THREAD

**Issue by lasers** [BasioMeusPuga/twitchy#27]
Hi. If you run this command twice with empty database, you get this. Thought you should know.
```bash
⚡ twitchy --non-interactive
Traceback (most recent call last):
  File "/usr/bin/twitchy", line 11, in <module>
    load_entry_point('twitchy==3.4.0', 'console_scripts', 'twitchy')()
  File "/usr/lib/python3.7/site-packages/twitchy/__main__.py", line 386, in main
    non_interactive('get_online')
  File "/usr/lib/python3.7/site-packages/twitchy/__main__.py", line 242, in non_interactive
    id_string_list = [str(i[0]) for i in channel_data]
TypeError: 'NoneType' object is not iterable
```

**Comment by BasioMeusPuga** [comment]
I'll fix this in the next commit.

**Comment by BasioMeusPuga** [comment]
Should be working now. Keep in mind that since this is `--non-interactive`, no output will be issued.

**Comment by lasers** [comment]
Works okay now in latest master / fixed via commit 647be5d. Many thanks. :-)
